<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputWindow</class>
 <widget class="QMainWindow" name="OutputWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>500</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btCompound">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>合成</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btPreView">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>预览</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btNext">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>180</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res/res.qrc">
      <normaloff>:/icon/next.png</normaloff>:/icon/next.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btPrevious">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>61</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res/res.qrc">
      <normaloff>:/icon/previous.png</normaloff>:/icon/previous.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseDirectory">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>460</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>选择目录</string>
    </property>
   </widget>
   <widget class="QLabel" name="useTime">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>460</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="pictureMap">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>821</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border-width: 1px;
border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>460</y>
      <width>219</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>文件名：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filename"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>.gif</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>500</y>
      <width>239</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>保存目录：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filedirectory">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btgetFilePath">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="res/res.qrc">
         <normaloff>:/icon/folder.png</normaloff>:/icon/folder.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>500</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>编码器：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Oct</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Neu</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/res.qrc"/>
 </resources>
 <connections/>
</ui>
